types/setting.go

```diff
@@ -156,4 +158,14 @@ var (
 		ReadOnly:    false,
 		Default:     "0",
 	}
+
+	SettingDefinitionDefaultLonghornStaticStorageClass = SettingDefinition{
+		DisplayName: "Default Longhorn Static StorageClass Name",
+		Description: "Set storageClassName when creating PV/PVC for existing volume. Users don't need to create related StorageClass since it is used as a tag for PVC bound. By default 'longhorn-static'.",
```

Comment:
```
"The 'storageClassName' for PV/PVC when creating PV/PVC for an existing Longhorn volume. Notice that it's unnecessary for users create the related StorageClass object in Kubernetes since the StorageClass would only be used as matching labels for PVC bounding purpose. By default 'longhorn-static'."
```